bot/cogs/tickets.py

```diff
@@ -0,0 +1,405 @@
+        return solved_tag
+
+    @is_ticket_or_dm()
+    @commands.hybrid_command(name="close", aliases=["solved", "closed", "resolved"])
+    async def close(self, ctx: RoboContext) -> None:
+        """Closes the thread"""
+        query = """
+        DELETE FROM tickets
+        WHERE thread_id = $1 AND owner_id = $2;
+        """
+        get_owner_id_query = """
+        SELECT owner_id
+        FROM tickets
+        WHERE thread_id = $1 AND location_id = $2;
+        """
+
+        async with self.pool.acquire() as conn:
+            if await self.can_close_ticket(ctx):
+                owner_id = ctx.author.id
+                if await self.can_admin_close_ticket(ctx):
+                    owner_id = await conn.fetchval(
+                        get_owner_id_query,
+                        ctx.channel.id,
+                        self.bot.transprogrammer_guild_id,
+                    )
+
+                await self.tick_post(ctx)
+                closed_ticket = await self.close_ticket(ctx.author, conn)
+                if closed_ticket is None:
+                    await ctx.send(
+                        "The ticket can not be found. Are you sure you have an open ticket?"
+                    )
+                    return
+                await conn.execute(query, closed_ticket.id, owner_id)
+                get_cached_thread.cache_invalidate(self.bot, owner_id, self.pool)
+                get_partial_ticket.cache_invalidate(self.bot, owner_id, self.pool)
+                await self.notify_finished_ticket(ctx, owner_id)
+
+    @is_ticket_or_dm()
+    @commands.hybrid_command(name="is-active", aliases=["is_active"])
+    async def is_active(self, ctx: RoboContext) -> None:
+        """Determines whether the current ticket is active"""
+        ticket = await get_cached_thread(self.bot, ctx.author.id, self.pool)
+        is_thread_active = ticket is not None
+        format_str = "Active" if is_thread_active else "Not Active"
+        await ctx.send(f"The current ticket is: {format_str}")
+
+    @is_ticket_or_dm()
+    @commands.hybrid_command(name="ticket-info", aliases=["tinfo"])
+    async def ticket_info(self, ctx: RoboContext) -> None:
+        """Provides information about the current ticket"""
+        ticket = await get_cached_thread(self.bot, ctx.author.id, self.pool)
+        partial_ticket = await get_partial_ticket(self.bot, ctx.author.id, self.pool)
+        if ticket is None or partial_ticket.id is None:
+            await ctx.send(
+                "You have no active tickets. Please send a message to Rodhaj to get started"
+            )
+            return
+
+        formatted_tags = ", ".join(
+            tag.name for tag in ticket.thread.applied_tags
+        ).rstrip(",")
+        ticket_owner = self.bot.get_user(partial_ticket.owner_id) or (
+            await self.bot.fetch_user(partial_ticket.owner_id)
+        )
+        embed = Embed()
+        embed.title = f"\U0001f3ab {ticket.thread.name}"
```

Comment:
```py
embed.title = f"\U0001f3ab {ticket.thread.name}" # U+1F3AB Ticket
```

Reply:
Fixed in commit adee012304df96d8a492c485c54dedcdd8a879bb